.eslintrc.js

```diff
@@ -102,8 +102,6 @@ module.exports = {
     'react/no-array-index-key': 'off',
     'react/no-danger': 'error',
     'react/no-direct-mutation-state': 'error',
-    // Required for backward compatibility. TODO v5, drop
-    'react/no-find-dom-node': 'off',
```

Comment:
Cool, I have created a note to remove that rule from the material-ui-x repository.